recipes/qwt/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(example example.cpp)
+# Must compile with "-fPIC" since Qt was built with -reduce-relocations.
+target_compile_options(example PRIVATE -fPIC)
```

Comment:
```suggestion
```
This should not be needed any more

Reply:
This is in a couple more recipes. IIRC, qwt is one of them

Reply:
I recently modified at recipe so that it's not needed

Reply:
Ok, sorry I did not notice that this recipe was using qt 5. The fix I had in mind is for qt 6 only currently.